### PR TITLE
webdriverxx: add package

### PR DESCRIPTION
**Pull request by Chi-EEE**
Selenium Server Version & Code to use this: https://github.com/GermanAizek/webdriverxx/issues/46

### CODE REVIEW

File: packages/l/libcurl/xmake.lua [hunk 1]
```diff
@@ -158,6 +158,8 @@ package("libcurl")
         handledependency("mbedtls", "mbedtls", "MBEDTLS_INCLUDE_DIRS", {MBEDTLS_LIBRARY = "mbedtls", MBEDX509_LIBRARY = "mbedx509", MBEDCRYPTO_LIBRARY = "mbedcrypto"})
         handledependency("zlib", "zlib", "ZLIB_INCLUDE_DIR", "ZLIB_LIBRARY")
         handledependency("zstd", "zstd", "Zstd_INCLUDE_DIR", "Zstd_LIBRARY")
+        io.writefile("GIT-INFO.md", "")
+        os.runv("buildconf" .. (is_plat("windows") and ".bat" or ""), {})
```
Comment (waruqi) [inline packages/l/libcurl/xmake.lua]: package:is_plat

File: packages/l/libcurl/xmake.lua [hunk 2]
```diff
@@ -158,6 +158,13 @@ package("libcurl")
         handledependency("mbedtls", "mbedtls", "MBEDTLS_INCLUDE_DIRS", {MBEDTLS_LIBRARY = "mbedtls", MBEDX509_LIBRARY = "mbedx509", MBEDCRYPTO_LIBRARY = "mbedcrypto"})
         handledependency("zlib", "zlib", "ZLIB_INCLUDE_DIR", "ZLIB_LIBRARY")
         handledependency("zstd", "zstd", "Zstd_INCLUDE_DIR", "Zstd_LIBRARY")
+
+        -- https://github.com/curl/curl/issues/6575
+        if package:is_plat("windows") and os.exists("buildconf.bat") then
```
Comment (waruqi) [inline packages/l/libcurl/xmake.lua:163]: os.isfile

**Comment by Chi-EEE** [comment]
The error: "fatal error C1083: Cannot open source file: 'C:\Users\runneradmin\AppData\Local\.xmake\cache\packages\2405\l\libcurl_1\8.7.1\source\build\src\tool_hugehelp.c': Permission denied" is random and manually generating the file does not fix the error.

**Comment by waruqi** [comment]
> The error: "fatal error C1083: Cannot open source file: 'C:\Users\runneradmin\AppData\Local.xmake\cache\packages\2405\l\libcurl_1\8.7.1\source\build\src\tool_hugehelp.c': Permission denied" is random and manually generating the file does not fix the error.

no this file in source, https://github.com/curl/curl/blob/curl-8_7_1/src/tool_hugehelp.c

**Pull request merged**